Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.ts

```diff
@@ -10,7 +10,13 @@ import { WindowRef } from 'src/app/services/windowRef.service';
 import { ColumnDefinitions } from 'src/app/shared/config/columnDefinitions';
 import { Person, PersonStatus } from 'src/app/shared/models';
 import { BookmarksState, ToggleBookmark } from 'src/app/shared/states';
+import { MatDialogConfig, MatDialog } from '@angular/material';
+import { IncorrectUserInformationComponent } from 'src/app/shared/dialogs/user-information/incorrect-user-information.component';
 
+export interface DialogData {
```

Comment:
🙈 Kannst du das Interface noch in den Dialog verschieben? Du nutzt es hier nicht, falls du ihm einen Sinn geben möchtest würde ich es wie folgt ändern:
````ts
dialogData: DialogData = {
        Firstname: this.person.Firstname,
        Id: this.person.Id
      }
const dialogConfig: MatDialogConfig = {
      autoFocus: true,
      data: dialogData
    };
````
somit hast du auch was von der Typ-Sicherheit :)
Falls du dich dafür entscheidest bitte den Typ noch umbenennen in `IncorrectUserInformationDialogData`